css-ui-4/Overview.bs

```diff
@@ -1541,6 +1541,11 @@ except:
 		where the computed value is always ''user-select/contain''
 		regardless of the specified value
 
+	<li>
+		on <a>locked editable element</a>s
+		where the computed value is ''user-select/contain''
+		if the specified value was ''user-select/text'' or ''user-select/auto''
```

Comment:
As far as the handling of `auto` for this case is mentioned below, it can probably be removed here.